source/browseMode.py

```diff
@@ -204,12 +204,36 @@ class BrowseModeTreeInterceptor(treeInterceptorHandler.TreeInterceptor):
 	def _get_currentNVDAObject(self):
 		raise NotImplementedError
 
+	ALWAYS_SWITCH_TO_PASS_THROUGH_ROLES = (

```

Comment:
Please make this a set (or a frozenset if you want it to be immutable) rather than a tuple. That makes checking membership a hash lookup instead of O(n) and this is not just a few elements. I realise it was a tuple before it was a constant. :)